src/server/game/Quests/QuestDef.h

```diff
@@ -134,33 +134,39 @@ enum QuestGiverStatus
-    QUEST_FLAGS_AUTO_TAKE               = 0x00200000,   // Automatically suggestion of accepting quest. Not from npc.
-    //QUEST_FLAGS_UNK2                    = 0x00400000,
-    //QUEST_FLAGS_UNK3                    = 0x00800000,   // Found in quest 14069
-    //QUEST_FLAGS_UNK4                    = 0x01000000,
-    // ... 4.x added flags up to 0x80000000 - all unknown for now
+    QUEST_FLAGS_NONE                        = 0x00000000,
+    QUEST_FLAGS_STAY_ALIVE                  = 0x00000001,   // Not used currently
+    QUEST_FLAGS_PARTY_ACCEPT                = 0x00000002,   // Not used currently. If player in party, all players that can accept this quest will receive confirmation box to accept quest CMSG_QUEST_CONFIRM_ACCEPT/SMSG_QUEST_CONFIRM_ACCEPT
+    QUEST_FLAGS_EXPLORATION                 = 0x00000004,   // Not used currently
+    QUEST_FLAGS_SHARABLE                    = 0x00000008,   // Can be shared: Player::CanShareQuest()
+    QUEST_FLAGS_HAS_CONDITION               = 0x00000010,   // Not used currently
+    QUEST_FLAGS_HIDE_REWARD_POI             = 0x00000020,   // Not used currently: Unsure of content
+    QUEST_FLAGS_RAID                        = 0x00000040,   // Can be completed while in raid
+    QUEST_FLAGS_TBC                         = 0x00000080,   // Not used currently: Available if TBC expansion enabled only
+    QUEST_FLAGS_NO_MONEY_FROM_XP            = 0x00000100,   // Not used currently: Experience is not converted to gold at max level
+    QUEST_FLAGS_HIDDEN_REWARDS              = 0x00000200,   // Items and money rewarded only sent in SMSG_QUESTGIVER_OFFER_REWARD (not in SMSG_QUESTGIVER_QUEST_DETAILS or in client quest log(SMSG_QUEST_QUERY_RESPONSE))
+    QUEST_FLAGS_TRACKING                    = 0x00000400,   // These quests are automatically rewarded on quest complete and they will never appear in quest log client side.
+    QUEST_FLAGS_DEPRECATE_REPUTATION        = 0x00000800,   // Not used currently
+    QUEST_FLAGS_DAILY                       = 0x00001000,   // Used to know quest is Daily one
+    QUEST_FLAGS_FLAGS_PVP                   = 0x00002000,   // Having this quest in log forces PvP flag
+    QUEST_FLAGS_UNAVAILABLE                 = 0x00004000,   // Used on quests that are not generically available
+    QUEST_FLAGS_WEEKLY                      = 0x00008000,
+    QUEST_FLAGS_AUTOCOMPLETE                = 0x00010000,   // Quests with this flag player submit automatically by special button in player gui
+    QUEST_FLAGS_DISPLAY_ITEM_IN_TRACKER     = 0x00020000,   // Displays usable item in quest tracker
+    QUEST_FLAGS_OBJ_TEXT                    = 0x00040000,   // use Objective text as Complete text
+    QUEST_FLAGS_AUTO_ACCEPT                 = 0x00080000,   // The client recognizes this flag as auto-accept.
+    QUEST_FLAGS_PLAYER_CAST_ON_ACCEPT       = 0x00100000,
+    QUEST_FLAGS_PLAYER_CAST_ON_COMPLETE     = 0x00200000,   // Automatically suggestion of accepting quest. Not from npc.
```

Comment:
you should remove this comment, its clearly wrong

Reply:
Done :)